be/src/util/spinlock.h

```diff
@@ -65,7 +65,7 @@ class SpinLock {
     // To try to get the best of everything, we will busy spin for a while before
     // yielding to another thread.
     // TODO: how do we set this.
-    static const int NUM_SPIN_CYCLES = 70;
+    static const int NUM_SPIN_CYCLES = 4000;
```

Comment:
Why more spins?

Reply:
> Why more spins?

The previous value is too conservative, and it is easy to cause unnecessary yields. I see that this value in many spinlock implementations are much larger than it, such as 4000 in folly:: MicroSpinLock, 1000 in absl: SpinLock and kudu